src/components/Hero.tsx

```diff
@@ -157,25 +178,41 @@ const Hero: React.FC<HeroProps> = ({ onClaim }) => {
           <Subtitle>
             The more UMA's TVL grows the more the KPI Options are worth!
           </Subtitle>
+          <Label>
+            <InputTitle level={3}>
+              Check if an address has KPI Options
+            </InputTitle>
+            <Input
+              placeholder="0x67C6Cf5288E5D8Bc474126949B3A50Cfe5512AF9"
```

Comment:
this is kind of confusing to see, do u think maybe this should be some text like paste address here